packages/fast-usdc/src/type-guards.js

```diff
@@ -50,7 +50,6 @@ export const CctpTxEvidenceShape = {
   },
   blockHash: EvmHashShape,
   blockNumber: M.bigint(),
```

Comment:
I wonder if we're ok with negative block numbers.

```suggestion
  blockNumber: M.nat(),
```